.github/workflows/integration-test.yml

```diff
@@ -35,30 +35,32 @@ jobs:
         with:
           go-version: 1.18
 
-      - name: Start 4 node docker cluster
-        run: make clean && INVARIANT_CHECK_INTERVAL=10 make docker-cluster-start &
+      - name: Setup Kurtosis repo
+        run:  echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+        
+      - name: Install Kurtosis
+        run: sudo apt-get update && sudo apt-get install -y kurtosis-cli
 
-      - name: Wait for docker cluster to start
-        run: |
-          until [ $(cat build/generated/launch.complete |wc -l) = 4 ]
-          do
-            sleep 10
-          done
-          sleep 10
+      - name: Build docker image
+        run: make clean && make build-docker-node
+      
+      - name: Run cluster with 4 nodes
+        run: >
+            kurtosis run github.com/kurtosis-tech/sei-package@b130bd658f9bf58e6410e93ace46398435df0d1c '{"cluster_size": 4, "num_accounts": 10, "invariant_check_interval": 10, "image": "sei-chain/localnode", "git_url": "${{ github.server_url }}/${{ github.repository }}/", "git_ref": "${{ github.head_ref}}"}' --enclave sei-integration-test
```

Comment:
How do we update the image in the future?

Reply:
Hey @yzang2019, this is a Starlark script with generic code to spin up a SEI blockchain. The code currently lives here https://github.com/kurtosis-tech/sei-package. We are more than happy to give you and team maintainer rights to that repo

Reply:
Hmm that seems not very convenient, we might need to constantly change the config on our local machine and launch the local docker based on that. We might also frequently change some configs to build the docker cluster

Reply:
For example, we need to frequently change the genesis step and tune some parameters: https://github.com/sei-protocol/sei-chain/blob/master/docker/localnode/scripts/step2_genesis.sh#L16

Reply:
It is definitely possibly to parametrize the genesis overrides so you can "inject" custom behavior when interacting with Kurtosis via command line or CI. Here is a concrete example to illustrate https://github.com/kurtosis-tech/sei-package/pull/8/files

Reply:
I see, is there any instruction for how to build and release a new image if we want to make some changes to the logic of building up and genesis a chain

Reply:
A starlark script is not an image like Docker's where you need to build and push to a specific repository. A starlark script is interpreted by Kurtosis, and fetched directly from Github. There is no need for ahead of time compilation. So to update the logic it is as easy as pushing the change to the repo. A couple of references:

https://docs.kurtosis.com/explanations/why-kurtosis-starlark
https://docs.kurtosis.com/concepts-reference/starlark

Reply:
Oh interesting, so basically we need to update Kurtosis repro if we want to make changes to the chain?

Reply:
If you're referring to the logic of building up and using a genesis state to start your chain, then yes! Kurtosis packages are written in Starlark, a dialect of Python and we would be available to support and guide you on how to write / edit the package. We can also grant maintainer permissions to you and your team so you can make your own changes at your leisure. 

Does that make sense @yzang2019 ?